tasks/pkg-redhat.yml

```diff
@@ -1,24 +1,36 @@
 ---
-- name: Download new RPM key
+- name: Download new RPM key (Expires in 2022)
   get_url:
     url: "{{ datadog_yum_gpgkey_e09422b3 }}"
     dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
     checksum: "sha256:{{ datadog_yum_gpgkey_e09422b3_sha256sum }}"
 
-- name: Import new RPM key
+- name: Import new RPM key (Expires in 2022)
   rpm_key:
     key: /tmp/DATADOG_RPM_KEY_E09422B3.public
     state: present
   when: not ansible_check_mode
 
+- name: Download new RPM key (Expires in 2024)
+  get_url:
+    url: "{{ datadog_yum_gpgkey_20200908 }}"
+    dest: /tmp/DATADOG_RPM_KEY_20200908.public
+    checksum: "sha256:{{ datadog_yum_gpgkey_20200908_sha256sum }}"
+
+- name: Import new RPM key (Expires in 2024)
+  rpm_key:
+    key: /tmp/DATADOG_RPM_KEY_20200908.public
+    state: present
+  when: not ansible_check_mode
+
 - name: Install Datadog yum repo
   yum_repository:
     name: "ansible_datadog_{{ item.key }}"
     description: Datadog, Inc.
     baseurl: "{{ item.value }}"
     enabled: yes
     gpgcheck: yes
-    gpgkey: "{% if datadog_agent_major_version|int == 7 %}{{ datadog_yum_gpgkey_e09422b3 }}{% else %}{{ datadog_yum_gpgkey }}{% endif %}"
+    gpgkey: "{% if datadog_agent_major_version|int == 7 %}{{ datadog_yum_gpgkey_20200908 }}{% else %}{{ datadog_yum_gpgkey }}{% endif %}"
```

Comment:
I don't know which one should be put here TBH. Is there a way to specify multiple keys here? If there is, we should put all keys there. If not, I _guess_ we should still put the currently used key (e09422b3) since that's the key people need right now. The steps above will ensure that the future key is present.

Reply:
If I understand correctly (and that's a big if)
- This line is actually not necessary due to the manual import steps above
- Since we're manually importing both keys above, it will work either way
- I chose the most recent key so that when the older key expires it doesn't cause problems